apps/wing/src/cli.ts

```diff
@@ -100,6 +102,27 @@ async function main() {
     }
   }
 
+  async function watchHook(cmd: Command) {
```

Comment:
I think handling the watch should be inside the test command itself. It may require some refactoring, but it has several benefits. One of the important ones is that it prevents the cli from loading `chokidar` unless needed.